docs/01-programs/01-token-metadata/04-token-standard.md

```diff
@@ -182,3 +182,57 @@ These are the "standard" non-fungible tokens the community is already familiar w
 
 </AccordionCode>
 </Accordion>
+
+## The Programmable Non-Fungible Standard
```

Comment:
We are missing the `ProgrammableNonFungible` on the list in "The Token Standard field" section.